### PR TITLE
Firu a dummy query at end of search handler constructor

### PR DESCRIPTION
**Pull request by jonmv**
@bratseth is this too naïve, or just perfect? :>

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java [hunk 1]
```diff
@@ -150,6 +152,24 @@ private static int examineExecutor(Executor executor) {
         return Integer.MAX_VALUE; // assume unbound
     }
 
+    private void warmup() {
+        try {
+            String uri = "/search/?yql=select+*+from+sources+*+where+true+limit+1;";
+            HttpRequest request = HttpRequest.createTestRequest(uri, com.yahoo.jdisc.http.HttpRequest.Method.GET);
+            Query query = new Query.Builder().setRequest(request)
```
Comment (jonmv) [inline container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java:159]: Hmm, perhaps this can be simpler ...

**Review by bratseth — APPROVED** [review]

**Comment by bratseth** [comment]
LGTM!

**Comment by aressem** [comment]
This broke a handful of system tests : https://factory.vespa.oath.cloud/testrun/63143128

**Comment by jonmv** [comment]
Yes, mostly metrics expecting 0 queries, but there was also one instance where the warmup query failed. I'll have a look at all of it.

**Pull request merged**